src/main/java/pl/jakubkozlowski/learning/firststeps/controller/descriptor/ChampionControllerDescriptor.java

```diff
@@ -1,7 +1,8 @@
 package pl.jakubkozlowski.learning.firststeps.controller.descriptor;
 
 public class ChampionControllerDescriptor {
-    public static final String BASE_PATH = "/champions";
+    public static final String BASE_PATH_CHAMPION = "/champions";
```

Comment:
wo,wo,wo. OOP! If there is ChampionControllerDescriptor, there should also be UserControllerDescriptor